### DISCUSSION TITLE
Apple's Music App (Formerly iTunes) Play Count Import

### DISCUSSION THREAD

**Discussion by lzilioli** [beetbox/beets#5071] [Q&A]
I have had my same iTunes library since 2006 and am very attached to the play count data that is in my Music app. Is there an easy way to import this play count data into the beets app?

**Comment by Andrey111113357** [comment]
Скачайте Outlook для iOS<https://aka.ms/o0ukef>
________________________________
От: Luke Zilioli ***@***.***>
Отправлено: Sunday, January 7, 2024 6:34:08 PM
Кому: beetbox/beets ***@***.***>
Копия: Subscribed ***@***.***>
Тема: [beetbox/beets] Apple's Music App (Formerly iTunes) Play Count Import (Discussion #5071)


I have had my same iTunes library since 2006 and am very attached to the play count data that is in my Music app. Is there an easy way to import this play count data into the beets app?
